cli/bench/types.go

```diff
@@ -16,15 +18,31 @@ type Results struct {
 	requestsPerSecond    int     // Cumber of requests per second - the main measured value.
 }
 
+// RotaryConnectionsPool describes round-cycled connection pool.
```

Comment:
For the future, I think you may consider developing bench as [`tt`](https://github.com/tarantool/tt) extension. `tt` uses [tarantool/go-tarantool](https://github.com/tarantool/go-tarantool) as a connector, and go-tarantool has built-in support of connection pools. It is also worth to mention that you should be able to develop an extention within your own repo.